Python/CMakeLists.txt

```diff
@@ -1,22 +1,14 @@
 ## Python module ##
 
-option(BUILD_PYTHON_INTERFACE "Build the Python interface" ON)
+option(INSTALL_PYTHON_INTERFACE "Build the Python interface" ON)
 
-find_program(PYTHON3_EXECUTABLE python3)
+find_package(Python3)
 
-if(BUILD_PYTHON_INTERFACE AND NOT APPLE AND PYTHON3_EXECUTABLE)
-      execute_process (
-           COMMAND ${PYTHON3_EXECUTABLE} -c
-           "import site, sys; sys.stdout.write(site.getsitepackages()[0])"
-           OUTPUT_VARIABLE PYTHON3_SITE_PACKAGES
-         )
-      SET(PYTHON3_MODULE_INSTALL_DIR ${PYTHON3_SITE_PACKAGES} CACHE
-       PATH "Python3 module install dir")
-       
-      if (PYTHON3_MODULE_INSTALL_DIR)
-         message(STATUS "PYTHON3 MODULE INSTALL DIR: ${PYTHON3_MODULE_INSTALL_DIR}" )
-        install(FILES ${CMAKE_CURRENT_SOURCE_DIR}/ctcsound.py
-            DESTINATION ${PYTHON3_MODULE_INSTALL_DIR})
-      endif()
+assign_bool(NOT_APPLE NOT APPLE)
+check_deps(INSTALL_PYTHON_INTERFACE NOT_APPLE Python3_FOUND)
```

Comment:
Why not on APPLE? The existing code doesn't seem to make that distinction?

Reply:
Well, there is a `AND NOT APPLE` in the original code, I have no idea why it's there though. We could potentially try to get rid of it and see what happens?